pull-requests.md

```diff
@@ -1,63 +1,148 @@
+### Comment
+This adds a comment to the PR at a selected position in the code display. The comment will be visible within the pull request both in the changes tab and the discussion tab.
+
+This should be the most common kind of review. Use it to discuss changes, to ask questions and to recommend most code changes.
+
+Adding a comment does not prevent merging the pull request. It is the PR owners responsibility to ensure that all comments in a PR are addressed and responded to before merging.
+
+### Request change
+A review that requests a code change will block any deployment of the PR.
+
+This will remain until the reviewer who requested the change approves the PR - even if another user reviews and approves the code change.  No one can merge this change until the reviewer approves it.
+
+This is a very “heavy-handed” action and should be reserved for situations where the requested change is needed to prevent significant problems.
+
+It is up to the developers discretion as to what constitutes a significant problem as this will be different in every code base, but some generic examples are:
+- Change will introduce a run-time error.
+- Change will introduce a performance problem.
+- Change will introduce a problem with other parts of the system.
+
+If a reviewer applies the “Request change” action, then they take ownership of getting that change applied. It is their responsibility to pair with and assist the developer to apply the fix they have requested.
+
+If the requested change is in dispute, for example, the author of the PR disagrees with the change or thinks it should be applied separately, the process for resolution is as follows:
+- PR author and change requester speak face-to-face (or on chat), and attempt to resolve their differences. The outcome of this discussion should be summarised and recorded in the PR.
+- If the PR author and change requester are unable to agree, a third party should be brought into the discussion to facilitate, and if necessary make a final decision on the item under consideration. The outcome of this discussion should be summarised and recorded in the PR.
 
+### Approve
+Approving the PR will allow the author to merge the code. The author should address, and respond to comments on the PR before they do so.
+
+PR approval allows the addition of a general comment that will be displayed in the conversation tab.
+
+Note: If a different reviewer has requested a code change using the “request change” option, you will not be able to approve the PR.  It is effectively blocked until the requester has approved the PR.
+
+# Merging is not the end!
+A PR is not finished when you merge it - check your PR in Production after it's deployed! Although unit & integration tests can add a lot of confidence that your change is good, looking at your change in Production is the ultimate ,and most important place to check your PR is delivering what's wanted.
```

Comment:
Rogue comma, mix of ampersand and not, use of capitalised Production, but PROD is used elsewhere.


```suggestion
A PR is not finished when you merge it - check your release in PROD after it's deployed! Although unit and integration tests can add a lot of confidence that your change is good, looking at your change in PROD is the ultimate and most important place to check your PR is delivering what's wanted.
```

Reply:
I missed some! https://github.com/guardian/recommendations/pull/110.